pipelines/datasets/us_sec_edgar/tasks.py

```
@@ -0,0 +1,136 @@
+    try:
+        frame = bd.read_sql(
+            f"select id_tabela, nome_coluna, chave "
+            f"from `{billing_project_id}.{DATASET_ID}.dicionario`",
+            billing_project_id=billing_project_id,
+            from_file=True,
+        )
+        published = [
+            {str(k): str(v) for k, v in record.items()}
+            for record in frame.to_dict("records")
+        ]
+    except Exception as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)